### DISCUSSION TITLE
Will Argon2id.ComputeHash always make use of the full buffer?

### DISCUSSION THREAD

**Discussion by kikaragyozov** [samuel-lucas6/Geralt#13] [Q&A] [answered]
If a buffer is allocated with a size of  `Argon2Id.MaxHashSize` and then passed to `Argon2Id.ComputeHash` as the `hash` parameter, can I assume that the full buffer size will be used by the underlying implementation, i.e a total of `Argon2Id.MaxHashSize` bytes will be written down?

**Comment by samuel-lucas6** [comment, marked as answer]
Good question. The actual output string may be shorter than that, but it will be padded with null characters, so you can treat it as that size.
